### DISCUSSION TITLE
I created a warp, it won't let me go, it tells me that I don't have access to that command

### DISCUSSION THREAD

**Discussion by facuuu122a** [EssentialsX/Essentials#5175] [Q&A] [answered]
I created a warp and it does not let me go, it tells me that I do not have access to that command, I looked for help on the internet but I did not receive anything, what do I do?

**Comment by Evidentsinger14** [comment, marked as answer]
You need to give yourself permission for the command. You can find a list of permissions [here](https://essinfo.xeya.me/permissions.html) 

If you need further assistance, you may join the support discord - https://discord.gg/PHpuzZS
